calm/control-example/one-node-wonder.json

```diff
@@ -8,21 +8,16 @@
       "node-type": "system",
       "name": "Example System",
       "description": "Example System",
-      "domains": {
-        "cbom": [
-          {
-            "unique-id": "ci-arch-001",
-            "description": "Evidence of Pre Production Review",
-            "requirement-schema": "control-specification.json",
-            "requirement-instance": "control-instance.json",
-            "evidence": [
+      "controls": {
+        "cbom": {
+            "base-path": ".",
+            "description": "Control requirements for delivering patterns",
+            "requirements": [
               {
-                "evidence-link": "https://evidence.com/12345",
-                "evidence-id": "12345"
+                "$ref": "pre-prod-review-configuration.json"
```

Comment:
I've been playing with this locally, once I fixed the refs link in the schema I got this to correctly recognise it is an array of control-requirements, but I can only enter them inline, at least in IntelliJ, using a $ref like this is not resolving, even though I think it should.

We introduced it as a way to effectively allow tools to look for configuration files all in the same place and therefore presumably be of the same 'domain'. However, I don't think this actually works. If we can get the external refs to work correctly and provide absolute URIs then that would enable architects to traverse to the actual instance (assuming the URL is physically accessible). 

Something to discuss further.